komodor/README.md

```diff
@@ -0,0 +1,31 @@
+# Komodor - Kubernetes Troubleshooting Platform
+
+## Overview
+
+This integration allows Datadog Monitor Alerts to be available in the Komodor Kubernetes troubleshooting platform and to suggest related services based on service connections detected by Datadog.
+
+Komodor tracks changes across your entire K8s stack, analyzes their ripple effect, and provides you with the context you need to troubleshoot efficiently and independently. Komodor gives you insight into your Kubernetes deployments on a timeline with the relevant information like what changed, what code was pushed and by who. You can also view data from your Git, config maps, infrastruture, alerting and other tooling such as Datadog, in one centralized and easy-to-understand display, easily link to Datadog metrics with dynamic deployment links that take you directly to the matrices you need. This allows you to troubleshoot your microservices based on the most relevant context, connections and service dependencies detected in Datadog.
+
+## Setup
+
+1. Log onto the Komodor platform [here][7].
+2. Install the Komodor pod-based agent on each Kubernetes cluster by using a Helm chart or Kustomize. For more information, see the [Komodor docs][2] for installing the agent.
+
+3. Once the agent is installed, set up the Datadog integrations listed below:
+    - [Komodor platform integration][3] -  This initial integration allows Datadog Monitor Alerts to be available in Komodor and will suggest related services based on service dependencies detected in Datadog. 
```

Comment:
Hi @angus-young, could you please explain the difference between the platform and webhook integrations?

Reply:
Hi @anawishnoff The first step provides us with the customers Datadog API and Token key and this is when we can discover service dependencies within Datadog, the customer would have to continue on to the Webhook setup in order for us to receive alerts. I've change the above to better reflect this.